src/components/main/EditorPicks.tsx

```
@@ -1,173 +1,149 @@
+        modifiers: {
+          x: gsap.utils.unitize(x => parseFloat(x) % -totalWidth)
+        }
+      })
+    }, 0)
+    // 마우스 오버 시 일시정지, 아웃 시 재생
+    const area = carouselAreaRef.current
+    const pause = () => anim && anim.pause()
+    const resume = () => anim && anim.resume()
+    if (area) {
+      area.addEventListener('mouseenter', pause)
+      area.addEventListener('mouseleave', resume)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)